authors/settings.py

```diff
@@ -146,3 +148,5 @@
     #     'authors.apps.authentication.backends.JWTAuthentication',
     # ),
 }
+django_heroku.settings(locals())
+
```

Comment:
Why add a blank line after every piece of code?

Reply:
Error on my part, I had misread the PEP 8 guidelines on [blank](https://www.python.org/dev/peps/pep-0008/#blank-lines)  lines.